### PR TITLE
DTFS2-5079 fixed null date on shouldCoverStartOnSubmission

### PR DESCRIPTION
**Pull request by Zainzzkk**

### CODE REVIEW

**Review by ttbarnes — COMMENTED** [review]
need a test

File: portal-api/api-tests/v1/gef/facilities.api-test.js [hunk 1]
```diff
@@ -289,7 +297,9 @@ describe(baseUrl, () => {
           required: [],
         },
       };
-
+      console.log('before', update);
+      console.log('bigexpected', expected);
+      console.log('after', expected.coverStartDate);
```
Comment (ttbarnes) [inline portal-api/api-tests/v1/gef/facilities.api-test.js:302]: remove logs please

File: portal-api/package.json [hunk 2]
```diff
@@ -12,7 +12,7 @@
     "start_azure_debug_fileshare": "node fileshare-test.js",
     "unit-test": "jest --notify --coverage --verbose --config=unit.jest.config.js ",
     "unit-test-quick": "jest --config=unit.jest.config.js --testTimeout=10000",
-    "api-test": "jest --notify --coverage --verbose --runInBand --config=api-test.jest.config.js",
+    "api-test": "jest --notify --verbose --runInBand --config=api-test.jest.config.js",
```
Comment (ttbarnes) [inline portal-api/package.json:15]: revert, coverage report is helpful 😝

File: portal-api/src/v1/gef/models/facilities.js [hunk 3]
```diff
@@ -150,9 +150,9 @@ class Facility {
         this.submittedAsIssuedDate = req.submittedAsIssuedDate;
       }
 
-      // nullify values based on previous questions
+      // sets to now date to now
```
Comment (ttbarnes) [inline portal-api/src/v1/gef/models/facilities.js:153]: sorry if it seems pedantic, but could we say something like "setting to midnight, we only care about the day". Just so it's totally clear for any future devs

File: portal-api/api-tests/v1/gef/facilities.api-test.js (hunk 1 above)
Comment (Zainzzkk) [reply portal-api/api-tests/v1/gef/facilities.api-test.js:302]: whoops, didn't press save before commit.  Removed

File: portal-api/package.json (hunk 2 above)
Comment (Zainzzkk) [reply portal-api/package.json:15]: and incredibly annoying :P reverted

File: portal-api/src/v1/gef/models/facilities.js (hunk 3 above)
Comment (Zainzzkk) [reply portal-api/src/v1/gef/models/facilities.js:153]: done

**Review by IgnatG — APPROVED** [review]

**Review by ttbarnes — APPROVED** [review]

**Pull request merged**